go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -432,13 +432,26 @@ func (c *Controller) addServiceToIdlingBalancer(vips sets.String, service *v1.Se
 	return nil
 }
 
+// RequestFullSync re-syncs every service that currently exists
+func (c *Controller) RequestFullSync() error {
+	klog.Info("Full service sync requested")
+	services, err := c.serviceLister.List(labels.Everything())
+	if err != nil {
+		return err
+	}
+	for _, service := range services {
+		c.onServiceAdd(service)
+	}
+	return nil
+}
+
 // handlers
 
-// onServiceUpdate queues the Service for processing.
+// onServiceAdd OnServiceUpdate queues the Service for processing.
```

Comment:
@trozet remove the "OnServiceUpdate" here?

Reply:
oops, done